packages/core/src/RenderPipeline/BasicRenderPipeline.ts

```diff
@@ -48,9 +47,10 @@ export class BasicRenderPipeline {
   constructor(camera: Camera) {
     this._camera = camera;
     const { engine } = camera;
-    this._cullingResults = new CullingResults(engine);
+    this._cullingResults = new CullingResults();
     this._cascadedShadowCaster = new CascadedShadowCasterPass(camera);
     this._depthOnlyPass = new DepthOnlyPass(engine);
+    this._spriteMaskManager = new SpriteMaskManager(camera.engine);
```

Comment:
Is it better to put `SpriteMaskManager` in `Scene`? Because :
1. Mask does not depend on rendering timing.
2. More memory efficient.